packages/serverless-contracts/vitest.config.ts

```diff
@@ -5,6 +5,7 @@ import { configDefaults, defineConfig } from 'vitest/config';
 export default defineConfig({
   plugins: [tsconfigPaths(), codspeedPlugin()],
   test: {
+    mockReset: true,
```

Comment:
Curious: why is that necessary now?

Reply:
It's not necessary, but I prefer to have the mocks reset before each test. It avoids side effects of one test on another. Otherwise, lots of `beforeEach` are needed to explicitly reset the mocks when some side effects are discovered, but some can be forgotten.

It's especially useful when counting mock calls. You want all the calls of the previous test to have been forgotten